src/daemonize.c

```diff
@@ -194,11 +201,11 @@ static void signal_handler(int signum) {
 }
 
 static int _set_signals(void) {
-    static char altstack[SIGSTKSZ];
+    static char altstack[RIBS_SIG_STACK_SIZE];
     static stack_t ss;
     memset(&ss, 0, sizeof(ss));
```

Comment:
this `memset` was also introduced with the code this is fixing but would seem to be unnecessary IMHO